rosidl_typesupport_connext_cpp/cmake/rosidl_typesupport_connext_cpp_generate_interfaces.cmake

```diff
@@ -129,5 +129,6 @@ install(
   DESTINATION "lib"
 )
 
-ament_export_include_directories(include)
 ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${Connext_LIBRARIES})
+
+ament_export_include_directories(include)
```

Comment:
Why moving this line around? The previous order (include before libraries) seems to make sense.